docker/.dockerignore

```diff
@@ -0,0 +1,7 @@
+Dockerfile
+.dockerignore
+node_modules
+npm-debug.log
+READMEs
+.next
```

Comment:
i think we'll want to include `.next` eventually when running `yarn build` / `yarn start`